kola/tests/kubernetes/controllerInstall.go

```diff
@@ -0,0 +1,795 @@
+package kubernetes
+
+const controllerInstallScript = `#!/bin/bash
+set -e
+
+# List of etcd servers (http://ip:port), comma separated
+export ETCD_ENDPOINTS=
+
+# Specify the version (vX.Y.Z) of Kubernetes assets to deploy
+export K8S_VER=v1.1.3
```

Comment:
I've parameterized this (locally) to single version variable in main. I'm not sure the best way to pass that into our tests since this will be run on os releases as well. 

Maybe we just maintain a list of the versions we want to test and run a cluster for each version?